index.html

```diff
@@ -1,23 +1,303 @@
 <!DOCTYPE html>
 <html lang="en">
```

Comment:
Use https://validator.w3.org/#validate_by_upload to check correctness of your HTML